pybop/plot/standard_plots.py

```diff
@@ -76,6 +76,7 @@ def __init__(
     ):
         self.x = x
```

Comment:
I think we can remove `self.x` and `self.y` from the class given the below changes.